_dev/gulp/settings/files.js

```diff
@@ -31,7 +31,10 @@ module.exports = {
   tmp: {
     css: [settings.path.tmp.css + '/**/*.css'],
     sass: [settings.path.tmp.sass + '/**/*.sass'],
-    js: [settings.path.tmp.js + '/**/*.js'],
+    js: {
```

Comment:
I think there will be issue with this.
`files` meant to be _minimatch_ pattern style for searching files. `path` is the one which contains paths as object map.